src/System.Security.Cryptography.Cng/pkg/System.Security.Cryptography.Cng.pkgproj

```diff
@@ -9,8 +9,11 @@
     <ProjectReference Include="..\ref\4.1\System.Security.Cryptography.Cng.depproj">
       <SupportedFramework>net461;netcore50</SupportedFramework>
     </ProjectReference>
+    <ProjectReference Include="..\ref\4.2\System.Security.Cryptography.Cng.depproj">
+      <SupportedFramework>netcoreapp1.0</SupportedFramework>
+    </ProjectReference>
```

Comment:
I'm given to understand by Eric that this isn't needed; but we can leave it for now.